src/test/java/net/pterodactylus/fcp/TempInputStreamTest.java

```diff
@@ -0,0 +1,54 @@
+package net.pterodactylus.fcp;

```

Comment:
This whole file uses CRLF as line separator and spaces instead of tabs. Do you want to fix that or should I do it?